packages/create-ship-app/scripts/full-stack.sh

```diff
@@ -40,6 +43,10 @@ echo ".idea" >> .gitignore
 echo "node-modules" >> .gitignore
 echo ".DS_Store" >> .gitignore
 
+# Copy docker-compose.yml
+
+cp "$cli_dir"/docker-compose/$docker_compose_file_name docker-compose.yml
```

Comment:
I think it will rewrite [this](https://github.com/paralect/ship/blob/ae769356a122db86a80c3b8d08d0a1f1fd01be19/services/api-koa/docker-compose.yml)  file if we are building only backend with koa